esp-hal/src/spi/master.rs

```diff
@@ -1448,9 +1448,21 @@ pub mod dma {
 
     #[cfg(feature = "async")]
     mod asynch {
+        use embedded_hal_async::spi::{ErrorType, SpiBus};
+
         use super::*;
 
-        impl<'d, T, C, M> embedded_hal_async::spi::SpiBus for SpiDma<'d, T, C, M, crate::Async>
+        impl<'d, T, C, M> ErrorType for SpiDma<'d, T, C, M, crate::Async>
```

Comment:
I think so - IIRC `embedded_hal_async::spi::ErrorType`  is a re-export of `embedded_hal::spi::ErrorType` - implementing it generic over mode in the `ehal1` module should be enough (since the `async` feature includes `embedded-hal` and `embedded-hal-async`)

Reply:
> embedded_hal_async::spi::ErrorType is a re-export of embedded_hal::spi::ErrorType

I see, I'll fix this now. Thank you all guys!